core/store/ledgerstore/ledger_store.go

```diff
@@ -864,6 +867,9 @@ func (this *LedgerStoreImp) saveCrossShardGovernanceData(block *types.Block, sha
 }
 
 func (this *LedgerStoreImp) saveParentShardConfig(block *types.Block) error {
+	if config.DefConfig.Genesis.ConsensusType == config.CONSENSUS_TYPE_SOLO {
```

Comment:
Is this ConsensusType for parent? or local?

Reply:
local, DefConfig is self config.